pages/Hazmat-Suit.md

```diff
@@ -0,0 +1,16 @@
+The Hazmat suit is part of the [Armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor) category.<br>
+It consists of four parts (listed below).<br><br>
+
+| Piece  |  Bonus effects  |
+| ------------- | -------------  |
+| Scuba Helmet  |  Water breathing  |
+| Hazmat Suit  |  Fire Resistance  |
+| Hazmat Suit Leggings |  N/A  |
+| Rubber Boots |  N/A  |
+
+## Obtaining
+Each piece can be crafted in the [Armor Forge](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Armor-Forge).
+
+## Usage
+This set is mostly used to provide protection against [Radiation](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Radiation).<br>
+The armour has poor durability, therefore it's best to only use it when necessary.
```

Comment:
*armor

(I know, I know, but let's be consistent on this page after writing it without an u for 2 times already...)

Reply:
Sorry, it's a force of habit to write it with a u, I went over it to remove the u's but I've skipped over this one :)